velox/vector/fuzzer/VectorFuzzer.h

```diff
@@ -33,6 +33,11 @@ enum UTF8CharList {
   MATHEMATICAL_SYMBOLS = 3 // Mathematical Symbols.
 };
 
+struct DataBoundary {
```

Comment:
nit: I think DataSpec is a better name since NaN and Infinity are not really the boundaries.